### DISCUSSION TITLE
Runner machine contains conflicting file -- can't pull

### DISCUSSION THREAD

**Discussion by dylan-bakr** [stefanzweifel/git-auto-commit-action#309] [Help] [answered]
not sure if this change to `server.R` is from the environment that we set using our GH_PAT or if this is some actual issue. seems like it's failing at the `_switch_to_repository` step in the main() function

![Screenshot (14)](https://github.com/stefanzweifel/git-auto-commit-action/assets/107076118/9e689a4b-743f-4991-8a07-be97761b853a)

**Comment by dylan-bakr** [comment, marked as answer]
seems like some overlap with #280  .. may be part of `actions/checkout@v3` issue?

**Reply by stefanzweifel** [reply]
I would agree, that this is similar to #280. But this is more of an internal _issue_ if git-auto-commit.
The Action [checks out](https://github.com/stefanzweifel/git-auto-commit-action/blob/master/entrypoint.sh#L102-L112) a new **local** branch which causes the issue you experience.

As [mentioned in #280](https://github.com/stefanzweifel/git-auto-commit-action/discussions/280#discussioncomment-4630572), you can either add `skip_checkout: true` to your workflow or add the following step to your workflow **before** you change any files.

```
- run: git checkout -b replace-me-with-the-right-branch-name
```

In a future major release of this Action, I will remove this "checkout local branch"-feature, as it causes a lot of issues for users.
